test/Transforms/affine-data-copy.mlir

```diff
@@ -107,3 +109,62 @@ func @matmul(%A: memref<4096x4096xf32>, %B: memref<4096x4096xf32>, %C: memref<40
 // CHECK:     dealloc [[BUFC]] : memref<128x128xf32>
 // CHECK:   }
 // CHECK: }
+
+//
+<<<<<<< HEAD
+// This test case will lead to single element buffers. These are eventually
+// expected to be turned into registers via alloca and mem2reg.
+=======
+// This test case will minimal fast mem capacity will lead to single element
+// memref buffers. These are eventually expected to be turned into registers via
+// alloca and mem2reg as the single iteration loops around them are cleaned
+// up.
+>>>>>>> 95a26422... Fix affine data copy generation corner cases/bugs
```

Comment:
Looks like there are some merge conflicts in here.

Reply:
Sorry about this - fixed.